include/swift/ClangImporter/CXXMethodBridging.h

```diff
@@ -103,6 +105,24 @@ struct CXXMethodBridging {
     // The first character is always lowercase.
     output.front() = std::tolower(output.front());
 
+    if (classifyNameKind() == NameKind::snake) {
+      for (std::size_t i = 0; i < output.size(); i++) {
+        size_t next = i + 1;
+        if (output[i] == '_') {
+          // If the first element is an underscore, remove it.
+          if (i == 0) {
+            output.erase(i, 1);
+          } else if (i < output.length()) {
```

Comment:
Sorry, I know we already talked about this, but should this be `if (next < output.length())` ?

Reply:
oh yeah, I think it should be `(next <= output.length())`